Index: bgl-python/boost/graph/detail/sparse_ordering.hpp
===================================================================
--- bgl-python/boost/graph/detail/sparse_ordering.hpp	(revision 338)
+++ bgl-python/boost/graph/detail/sparse_ordering.hpp	(working copy)
@@ -122,7 +122,7 @@
     public:      
       typedef typename Sequence::iterator iterator;
       typedef typename Sequence::reverse_iterator reverse_iterator;
-      typedef queue<Tp,Sequence> queue;
+      // typedef queue<Tp,Sequence> queue;
       typedef typename Sequence::size_type size_type;
 
       inline iterator begin() { return this->c.begin(); }
